Patch migrated from desktop gate, where it was listed as:
# date:2014-06-04 owner:henzhao type:bug bugdb:18057681

which corresponds to:
18057681 ck-seat-tool -a only accepts one variable

diff --git a/tools/ck-seat-tool.c.orig b/tools/ck-seat-tool.c
index 347c81c..773afb7 100644
--- a/tools/ck-seat-tool.c.orig
+++ b/tools/ck-seat-tool.c
@@ -184,7 +184,7 @@ add_session (DBusGConnection  *connection)
                                            (GDestroyNotify) g_free);
 
         if (remaining_args) {
-                for (i = 0; i < G_N_ELEMENTS (remaining_args); i++) {
+                for (i = 0; i < g_strv_length (remaining_args); i++) {
                         char **arr;
 
                         /* split var=value */

